src/components/draw/feature-edit-style.vue

```diff
@@ -24,20 +24,20 @@ const styleComponents = {
 //   ref(undefined)
 
 const currentStyleComponent = computed(() => {
-  const geomType = props.feature.geom
-  if (geomType === 'Point') {
+  const featureType = props.feature.featureType
```

Comment:
This breaks the UI: the component for styling is missing and the icon for the "Etiquette"/label is missing 
![image](https://github.com/user-attachments/assets/de22af3d-06cb-4443-ab81-6682addec545)

Reply:
There is no icon for Labels

Reply:
in production:
![image](https://github.com/user-attachments/assets/e3add98b-3788-4c09-bd04-cf88e1bf2c81)